src/charm.py

```diff
@@ -181,24 +193,30 @@ def _configure_workload(self, event: ops.charm.EventBase) -> None:
         statsd_container = self.unit.get_container("statsd-prometheus-exporter")
         # pebble may not be ready, in which case we just return
         if not gunicorn_container.can_connect() or not statsd_container.can_connect():
-            self.unit.status = MaintenanceStatus('waiting for pebble to start')
-            logger.debug('waiting for pebble to start')
+            self.unit.status = MaintenanceStatus("waiting for pebble to start")
+            logger.debug("waiting for pebble to start")
             return
 
-        logger.debug("About to add_layer with pebble_config:\n{}".format(yaml.dump(gunicorn_pebble_config)))
+        logger.debug(
+            "About to add_layer with pebble_config:\n{}".format(yaml.dump(gunicorn_pebble_config))
```

Comment:
I know this is just a change introduced by linting, it is probably worth just updating the logging to not use `.format` or `f-strings` as it is a security risk, feel free to do here or in a follow up PR although with some urgency: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#formatting-log-messages

Reply:
There are no new tests because the coverage was already at 99%. I'm changing the logging and pushing a new commit :)